examples/cfg.ron

```diff
@@ -40,42 +55,13 @@
             KEY_CAPSLOCK:  Tap(Key(KEY_LEFTCTRL)),
             KEY_LEFTCTRL:  Tap(Key(KEY_LEFTCTRL)),
 
-            // F1 on regular tap, layer toggle on hold.
-            // A toggled layer will stay active until toggled again.
-            KEY_F1:  TapHold(Key(KEY_F1), ToggleLayerAlias("f_is_enter")),
-
-            // F2 on regular tap, layer toggle by index otherwise.
-            KEY_F2:  TapHold(Key(KEY_F2), ToggleLayer(2)),
-
-            // F2 on regular tap, layer toggle by index otherwise.
-            KEY_F3:  TapHold(Key(KEY_F3), ToggleLayerAlias("h_is_enter")),
-
-            // F4 on regular tap, layer toggle by index otherwise.
-            KEY_F4:  TapHold(Key(KEY_F4), ToggleLayerAlias("i_is_enter")),
+            KEY_F1:  TapHold(Key(KEY_F1), Multi([DeactivateProfile("not-firefox"), ActivateProfile("firefox")])),
```

Comment:
With my above suggestion, I think these could be a single ActivateProfile effect.